tests/Type/Symfony/data/extension/with-configuration/WithConfigurationExtension.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace PHPStan\Type\Symfony\Extension\WithConfiguration;
+
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use \Symfony\Component\DependencyInjection\Extension\Extension;
+
+class WithConfigurationExtension extends Extension
+{
+	public function load(array $configs, ContainerBuilder $container): void
+	{
+		\PHPStan\Testing\assertType(
+			'PHPStan\Type\Symfony\Extension\WithConfiguration\Configuration',
```

Comment:
You could do `Configuration::class` here.

Reply:
Will do. I think I based it on some other test fixture, which also used a string. But if `::class` works and is preferred I will obviously change it 👍🏻